src/components/header/Header.jsx

```diff
@@ -11,15 +11,16 @@ import Breadcrumb from './Breadcrumb';
  * @extends React.Component
  */
 export default class Header extends Component {
-  state = {
-    ...this.initialState,
-  };
-
+  constructor(props) {
+    super(props);
+    this.state = {
+      ...this.initialState,
```

Comment:
why not just put the contents of `initialState` in `this.state`

Reply:
Ok @joykare  will do